### ISSUE TITLE
Links on the primary data are not serialized to JSON

### ISSUE THREAD

**Issue by nikolafon** [jasminb/jsonapi-converter#85]
When you add links to the primary data and try to serialize, links are missing in resulting JSON.

**Comment by jasminb** [comment]
Make sure you enable appropriate `SerializationFeature`:

```
converter.enableSerializationOption(SerializationFeature.INCLUDE_LINKS);
```

**Comment by nikolafon** [comment]
Yes. That is set but it still doesn't work.

**Comment by jasminb** [comment]
Hey @nikolafon, can you please provide example of how to reproduce the problem.

**Comment by nikolafon** [comment]
Try serialize java object with` @Links` annotated field, to JSON.
I think the `ResourceConverter` tries to set links data node but fails.
`dataNode.set(LINKS, links.get(LINKS));` but the `links` is the object to set.

**Comment by jasminb** [comment]
So, object that is being serialised has this:

```java
	@Links
	private com.github.jasminb.jsonapi.Links links;
```
?

**Comment by nikolafon** [comment]
Yes, the annotation is there.

**Comment by jasminb** [comment]
Is the correct type used?

Problem is that I cannot reproduce the issue, unit tests cover this functionality and it is working as intended.

See `SerializationTest.testFullSerialization()` for more details.

**Comment by jasminb** [comment]
Fixed in d074a91bc5569ef10e7f8e62f34687f774fb4ad2.
